007_smoothed_Laplace/smoothed_laplace.ipynb

```diff
@@ -0,0 +1,598 @@
+      "text/plain": [
+       "<Figure size 640x480 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "laplace = cuqi.distribution.Laplace(np.array([0]), np.array([1]))\n",
+    "smoothed_laplace = LaplaceSmoothed(location = np.array([0]), scale=1, beta=1e-3)\n",
+    "x = np.linspace(-5, 5, 100)\n",
+    "smoothed_laplace_pdf_values = np.zeros(x.shape)\n",
+    "laplace_pdf_values = np.zeros(x.shape)\n",
+    "smoothed_laplace_grad_values = np.zeros(x.shape)\n",
+    "laplace_grad_values = np.zeros(x.shape)\n",
+    "dx = 1e-4\n",
+    "for i in range(len(x)):\n",
+    "    smoothed_laplace_pdf_values[i] = smoothed_laplace.logpdf(x[i])\n",
+    "    laplace_pdf_values[i] = laplace.logpdf(x[i])[0]\n",
+    "    smoothed_laplace_grad_values[i] = smoothed_laplace.gradient(x[i])[0]\n",
+    "    laplace_grad_values[i] = (laplace.logpdf(x[i])[0]-laplace.logpdf(x[i]-dx)[0])/dx\n",
+    "\n",
+    "plt.figure()\n",
+    "plt.subplot(1,2,1)\n",
```

Comment:
PLease make (2,1,1) instead.  In the current 1x2 layout, the figures are squashed and the legend concealing the interesting part of the signal.

Reply:
Done